app/src/main/java/com/sopt/now/compose/data/module/ApiFactory.kt

```diff
@@ -0,0 +1,42 @@
+package com.sopt.now.compose.data.module
+
+import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
+import com.sopt.now.compose.BuildConfig
+import com.sopt.now.compose.data.network.AuthService
+import com.sopt.now.compose.data.network.FollowerService
+import kotlinx.serialization.json.Json
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+
+
+object ApiFactory {
+    private const val BASE_URL: String = BuildConfig.AUTH_BASE_URL
+    private const val FOLLOWER_URL: String = BuildConfig.FOLLOWER_URL
+
+    private val interceptorClient = OkHttpClient().newBuilder()
+        .addInterceptor(AuthInterceptor()).build()
+
+    val baseRetrofit: Retrofit by lazy {
+        Retrofit.Builder()
+            .baseUrl(BASE_URL)
+            .client(interceptorClient)
+            .addConverterFactory(Json.asConverterFactory("application/json".toMediaType()))
+            .build()
+    }
+
+    val followerRetrofit: Retrofit by lazy {
+        Retrofit.Builder()
+            .baseUrl(FOLLOWER_URL)
+            .addConverterFactory(Json.asConverterFactory("application/json".toMediaType()))
+            .build()
+    }
+
+    inline fun <reified T> createBase(): T = baseRetrofit.create(T::class.java)
+    inline fun <reified T> createFollower(): T = followerRetrofit.create(T::class.java)
```

Comment:
동료개발자가 createBase, createFollower 라는 네이밍을 보고, Retrofit 객체를 만드는걸 알 수 있을까요??
```suggestion
    inline fun <reified T> createBaseRetrofit(): T = baseRetrofit.createRetrofit(T::class.java)
    inline fun <reified T> createFollower(): T = followerRetrofit.create(T::class.java)
```

Reply:
헉 네이밍 제안 너무 감사해요 섬세한 개발자가 되게씁니다!!!!!